src/shared/components/ChannelPreviewBase/ChannelPreviewBase.style.tsx

```diff
@@ -2,17 +2,55 @@ import { css } from '@emotion/react'
 import styled from '@emotion/styled'
 import { Link } from 'react-router-dom'
 
-import { colors, sizes, transitions, typography } from '../../theme'
+import { colors, sizes, square, transitions, typography } from '../../theme'
 import { Avatar } from '../Avatar'
+import { Button } from '../Button'
 import { Text } from '../Text'
 
 const imageTopOverflow = '2rem'
 const containerPadding = '22px'
 
-export const OuterContainer = styled.article`
-  display: flex;
+export const OuterContainer = styled.article<{ variant?: string }>`
```

Comment:
This is not well aligned:
yours:
![image](https://user-images.githubusercontent.com/51168865/128136069-7fbc3fb9-2d59-4d80-863f-bfd7e6f930df.png)
design: 
![image](https://user-images.githubusercontent.com/51168865/128136168-7081616a-e18f-45ec-bbd7-76c3338a366e.png)

Reply:
`align-items: flex-start;` on `Info` should do the trick